rct229/rulesets/ashrae9012019/data_fns/table_G3_9_3_fins.py

```diff
@@ -0,0 +1,47 @@
+import math
+from typing import TypedDict
+
+from pint import Quantity
+from rct229.rulesets.ashrae9012019.data import data
+from rct229.rulesets.ashrae9012019.data_fns.table_utils import find_osstd_table_entry
+from rct229.schema.config import ureg
+from rct229.utils.assertions import assert_
+
+
+class FullLoadMotorEfficiency(TypedDict):
+    full_load_motor_efficiency_for_modeling: float
+
+
+def table_G3_9_3_lookup(shaft_input_power: Quantity) -> FullLoadMotorEfficiency:
+    """Returns the full-load motor efficiency for motors as required by ASHRAE 90.1 Table G3.9.2
+    Parameters
+    ----------
+    shaft_input_power: Quantity
+        shaft input power value
+
+    Returns
+    -------
+    dict
+        {full_load_motor_efficiency_for_modeling - The full load motor efficiency for modeling by Table G3.9.3}
+
+    """
+
+    shaft_input_power_mag = shaft_input_power * ureg("hp")
```

Comment:
Is it unit conversion? I do not quite understand this line.
The variable name suggest that this line is extracting the magnitude of the `shaft_input_power`. However, the right hand side is transforming the `shaft_input_power` to a `Quantity` object when it is already a `Quantity` object?

Reply:
Great catch! I just fixed this correctly.